test/check-services

```diff
@@ -65,6 +65,10 @@ Description=Test Template for %I
 ExecStart=/usr/bin/test-service %I
 """)
 
+        # Only root can read the system.journal on RHEL 7.
+        if m.os == "rhel-7":
+            b.default_user = "root"
```

Comment:
https://bugzilla.redhat.com/show_bug.cgi?id=1211499:

> Access list granting read access on /var/log/journal to members of wheel group is adjusted at package install time. In RHEL there is no /var/log/journal by default so after user creates it has to setup access list on directory manually.

We create /var/log/journal explicitly, but don't adjust the permissions, apparently. Too obvious...